src/components/Dashboard/Form/Applicants/Applicant.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../../scss/_mixins.scss';
+.Applicant{
+  @include flex(wrap);
+  width: 650px ;
+}
+button.developedDiv {
+  background-color: $orange;
+  border: none;
+  border-radius: 50%;
+  color: $white;
+  cursor: pointer;
+  font-size: $font-lg;
+  font-weight: $bold;
+  margin: 30px 0;
+  outline: none;
+  padding: 0 9px ;
+}
+label.labelForAdd {
```

Comment:
the same -> use only class ;) don't mix that with tag, css modules give us the unique classes anyway

Reply:
done